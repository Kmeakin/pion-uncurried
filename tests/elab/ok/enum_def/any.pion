enum Any {
    MkAny(A: Type, a: A),
}

let mk_any = MkAny;