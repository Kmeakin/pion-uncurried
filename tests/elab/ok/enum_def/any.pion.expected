enum Any: Type {
    MkAny(A: Type, a: A): Any,
}

let mk_any: fn(A: Type, a: A) -> Any = MkAny;
